src/components/volume/_volume.scss

```diff
@@ -1,6 +1,6 @@
 @import '~styles/variables';
 
-.control-button-container.volume-control {
+.control-button-container.control-volume {
```

Comment:
This is breaking change. There are customers override the volume style by this class

Reply:
Fixed.
BTW, checked on customers sites, they are all overriding the icon classes (`playkit-icon-volume` for example) so this ain't break anything. But anyway, for us to be on the safe side, fixed.